docs/src/main/tut/symbols.md

```diff
@@ -0,0 +1,19 @@
+| Symbol | Name                   | Typeclass      | Import                       | Source                                              |
+| ------ | ---------------------- | ------------   | --------------------------   | -------------------------------------------------   |
+| `<*>`  | apply                  | `Apply`        | TBA                          | core/src/main/scala/cats/Apply.scala                |
+| `|@|`  | Cartesian builder      | `Cartesian`    | `cats.syntax.cartesian._`    | core/src/main/scala/cats/Cartesian.scala            |
```

Comment:
OK, I'm fine with links to the files. In the case of generated sources, maybe we should italicize the name instead of linkifying it, and then putting a note at the bottom that italicized names are from generated source?